.travis.yml

```diff
@@ -8,7 +8,7 @@ go:
 
 install:
   - go get -d -t github.com/opentracing/opentracing-go/...
-  - go get -u github.com/golang/lint/...
```

Comment:
don't remove golint, it's a tool, not a dependency

Reply:
Hmmm are we sure? From reading the [dep documentation](https://golang.github.io/dep/docs/Gopkg.toml.html#required) it seemed like this is what `required` is for:

> Use this for: linters, generators, and other development tools that
> - Are needed by your project
> - Aren't imported by your project, directly or transitively
> - You don't want to put them in your GOPATH, and/or you want to lock the version